client/devdocs/design/playground.jsx

```diff
@@ -88,6 +88,9 @@ import JetpackColophon from 'components/jetpack-colophon';
 import JetpackLogo from 'components/jetpack-logo';
 import LanguagePicker from 'components/language-picker';
 import ListEnd from 'components/list-end';
+import NavItem from 'components/section-nav/item';
```

Comment:
I've noticed that these changes are duplicated across many PRs (#24231, #24229, #24225 and #24224), maybe it's a good idea to extract those into it's own PR 😉

Reply:
Just found your response on #24223 (https://github.com/Automattic/wp-calypso/pull/24223#discussion_r182001584). Sorry for the duplication 🙂

Reply:
Cool, I'll look at splitting those out.